pkg/apis/policy/v1alpha1/well_known_constants.go

```diff
@@ -9,4 +9,14 @@ const (
 
 	// ClusterPropagationPolicyLabel is added to objects to specify associated ClusterPropagationPolicy.
 	ClusterPropagationPolicyLabel = "clusterpropagationpolicy.karmada.io/name"
+
+	// NamespaceAutoPropagationLabel is added to namespace objects to indicate if
+	// the namespace should be skipped from propagating by the namespace controller.
+	// For example, a namespace with the following label will be skipped:
+	//   labels:
+	//     namespace.karmada.io/skip-auto-propagation: true
+	//
+	// NOTE: If create a ns without this label, then patch it with this label, the ns will not be
+	// synced to new member clusters, but old member clusters still have it.
+	NamespaceAutoPropagationLabel = "namespace.karmada.io/skip-auto-propagation"
```

Comment:
```suggestion
	NamespaceSkipAutoPropagationLabel = "namespace.karmada.io/skip-auto-propagation"
```
I changed my mind, this should be a skip label.